ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -34,6 +37,11 @@ function prepare_next_steps() {
   fi
 }
 
+function update_config_map() {
+  echo "Updating Cluster Infra ConfigMap with the start and end times."
+  ./oc patch configmap openshift-install -p "{'data':{'install_times':{'start_time':'$1','end_time':'$2'}}}"
```

Comment:
Specify the namespace.
```suggestion
  ./oc patch configmap openshift-install -n openshift-config -p "{'data':{'install_times':{'start_time':'$1','end_time':'$2'}}}"
```